framework/test/functional/mysql.test.js

```diff
@@ -19,7 +19,7 @@ const {
 	commitDbTransaction,
 	getDbConnection,
 	rollbackDbTransaction,
-} = require('../../shared/indexdb/mysql');
+} = require('../../src/mysql');
```

Comment:
Please add tests for primary key and composite primary key validation after the table is created.